### PR TITLE
add .cpu() to the returned value of render_rays()

### PR DESCRIPTION
**Pull request by johnnylu305**
# Issue
To calculate the psnr, it looks like the program will store all the reconstructed images and ground truth in the GPU until the end of the inference. This may suffer from OOM if the user's GPU memory is limited or the user's testing set contains several high-resolution images. 
# Solution
We can move them to CPU memory by .cpu() or even .detach().cpu() before we return the value from the render_rays(). This is similar to what you have done [here](https://github.com/kakaobrain/NeRF-Factory/blob/main/src/model/plenoxel/model.py) for plenoxel.

### CODE REVIEW

**Comment by jeongyw12382** [comment]
Thanks for your suggestion. We'll double-check whether your nice code works fine on our environment to avoid bugs. We'll soon let you know our decision.
